src/react-app-env.d.ts

```diff
@@ -1 +1,51 @@
 /// <reference types="react-scripts" />
+
```

Comment:
For small project you can keep all interfaces in one file. However it is good idea to split them into separated files in **types** dir. It makes live easier when project is big.